src/codeu/chat/common/VersionInfo.java

```diff
@@ -0,0 +1,46 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import codeu.chat.util.Uuid;
+
+/**
+ * Holds the current version of the CodeU chat server.
+ * 
+ * For clients, this can be used to determine whether or not the server version
+ * matches that of the client. For servers, this is used to advertise the
+ * version of the server.
+ */
+public class VersionInfo {
+  private static final String CURRENT_VERSION = "1.0.0";
+
+  private Uuid version;
+
+  /**
+   * Creates a new VersionInfo object with the current supported version.
+   */
+  public VersionInfo() {
+    try {
+      this.version = Uuid.parse(CURRENT_VERSION);
+    } catch (IOException e) {
+      this.version = null;
```

Comment:
Are you saying to invoke `Uuid.parse("1.0.0")` instead? The compiler will expect me to catch the IOException, but not possible inline. I could try making a `private static final Uuid getCurrentVersionUuid()` and then pass that to the default constructor.